webpack/components/GeneralTabContent.js

```diff
@@ -22,7 +22,7 @@ const GeneralTabContent = ({
       required
       type="number"
       value={general?.vmid?.value}
-      disabled={fromProfile}
+      disabled={!newVm || fromProfile}
```

Comment:
for better tracking, we should have own PRs to fix different issues and to have commit message what the purpose of the PR / commit is.